blogs/blocks/header/header.css

```diff
@@ -362,3 +362,7 @@ header .search-nav {
 header .search-nav[aria-expanded='true'] {
   display: block;
 }
+
+.appear {
```

Comment:
I think this ends up getting removed if we only load the one header, but the css scoping here should also start with a .header e.g. `.header.appear` to avoid affecting any styling outside of this block